app/com/arpnetworking/metrics/portal/alerts/impl/PluggableAlertRepository.java

```diff
@@ -272,6 +284,17 @@ private void reload(final InputStream stream) {
         _alerts = mapBuilder.build();
         _periodicMetrics.recordCounter(RELOAD_SUCCESS_COUNTER, 1);
         _periodicMetrics.recordGauge(RELOAD_GAUGE, _alerts.size());
+
+        // Immediately kick the coordinator to pick up any job changes so that we
+        // don't have to wait for anti-entropy to begin.
+        //
+        // NOTE: Since nodes have their own copy of this repository, this will
```

Comment:
Wait, so when is the coordinator actor ref not provided?

Reply:
as mentioned above, this was for testing.